content/en/news/releases/1.5.x/announcing-1.5.1/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Announcing Istio 1.5.1
+linktitle: 1.5.1
+subtitle: Patch Release
+description: Istio 1.5.1 patch release.
+publishdate: 2020-03-25
+release: 1.5.1
+aliases:
+    - /news/announcing-1.5.1
+---
+
+This release contains bug fixes to improve robustness and fixes for the security vulnerabilities described in [our March 25th, 2020 news post](/news/security/istio-security-2020-004). This release note describes what’s different between Istio 1.5.0 and Istio 1.5.1.
+
+{{< relnote >}}
+
+## Security update
+
+- **ISTIO-SECURITY-2020-004** Istio uses a hard coded `signing_key` for Kiali.
+
+__[CVE-2020-1764](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1764)__: Istio uses a default `signing key` to install Kiali. This can allow an attacker with access to Kiali to bypass authentication and gain administrative privileges over Istio.
+In addition, another CVE is fixed in this release, described in the Kiali 1.15.1 [release](https://github.com/kiali/kiali/releases/tag/v1.15.1).
```

Comment:
This link is wrong. Should be https://kiali.io/news/security-bulletins/kiali-security-001/